pkg/controller/eventing/inmemory/clusterchannelprovisioner/reconcile.go

```diff
@@ -127,6 +130,14 @@ func (r *reconciler) reconcile(ctx context.Context, ccp *eventingv1alpha1.Cluste
 		return nil
 	}
 
+	// The name of the svc has changed since version 0.2.1. Hence, delete old dispatcher service (in-memory-channel-clusterbus)
+	// that was created previously in version 0.2.0 to ensure backwards compatibility.
+	err := r.deleteOldDispatcherService(ctx, ccp)
```

Comment:
I'm not sure how much it matters because Istio won't notice right away, but move this after `util.CreateDispatcherService`, so that the correct `VirtualService` is updated before the old `Service` is deleted.

Reply:
I don't think it matters, I changed the order.